docker-compose.yml

```diff
@@ -148,17 +136,14 @@ services:
 
   flowauth:
     container_name: flowauth
-    image: flowminder/flowauth:latest
-    build:
-      context: ./flowauth
-      dockerfile: Dockerfile
+    image: flowminder/flowauth:${CONTAINER_TAG:-latest}
     ports:
       - ${FLOWAUTH_PORT:?Must set FLOWAUTH_PORT env var}:80
     environment:
       DEMO_MODE: ${DEMO_MODE:?Must set DEMO_MODE env var}
 
   query_locker:
-    container_name: redis_flowkit
+    container_name: query_locker
```

Comment:
Not for this PR, but perhaps we can rename `query_locker` to `flowmachine_query_locker` (or `flowkit_query_locker`)?